pkg/synchronization/controller.go

```diff
@@ -57,13 +58,15 @@ const (
 )
 
 const (
-	serviceCentralPollPeriod            = 30 * time.Second
-	serviceCentralListAllPeriod         = 30 * time.Minute
-	serviceCentralListDriftCompensation = 5 * time.Second
-	releaseManagementPollPeriod         = 5 * time.Second
-	releaseManagementSyncAllPeriod      = 30 * time.Minute
-	baseDelayProcSec                    = 15
-	rmsPollJitterFactor                 = 1.2
+	serviceCentralPollPeriod                               = 30 * time.Second
+	serviceCentralListAllPeriod                            = 30 * time.Minute
+	serviceCentralListDriftCompensation                    = 5 * time.Second
+	releaseManagementPollPeriod                            = 5 * time.Second
+	releaseManagementSyncAllPeriod                         = 30 * time.Minute
+	baseDelayProcSec                                       = 15
+	rmsPollJitterFactor                                    = 1.2
+	secretTypeOpsGenie                  core_v1.SecretType = voyager.Domain + "/opsgenie"
```

Comment:
Does this actually work? I'm not sure kube will like custom secret types.

Reply:
I think we use "opaque" type everywhere (default), but if it works - why not

Reply:
This is not yet tested. 🤷‍♂️ Should I change it to "opaque"?